### ISSUE TITLE
Create Docstrings

### ISSUE THREAD

**Issue by SaverioVad** [SaverioVad/HAD_Net#1]
Create docstrings for all functions

**Comment by WSake** [comment]
Can you share the following files?
A list containing the names (i.e., ids) of the samples used in the training set, saved as a .npy file.
A list containing the names (i.e., ids) of the samples used in the (local) validation set, saved as a .npy file.
A list containing the names (i.e., ids) of the samples used in the (local "held-out") testing set, saved as a .npy file, if one is used.

Thanks.

**Comment by SaverioVad** [comment]
Rather than providing these files in the repo, I decided to leave the task of splitting the data into a training, validation, and testing set up to the discretion of the user. These files can be easily generated by simply traversing the directory containing the training data and randomly assigning a set label to each sample. 

Hope this helps!

**Comment by WSake** [comment]
> Rather than providing these files in the repo, I decided to leave the task of splitting the data into a training, validation, and testing set up to the discretion of the user. These files can be easily generated by simply traversing the directory containing the training data and randomly assigning a set label to each sample.
> 
> Hope this helps!

Ok, thanks for your reply.

**Comment by 1249402434** [comment]
> Rather than providing these files in the repo, I decided to leave the task of splitting the data into a training, validation, and testing set up to the discretion of the user. These files can be easily generated by simply traversing the directory containing the training data and randomly assigning a set label to each sample.
> 
> Hope this helps!

Sorry, I'm not competent enough. I still hope you can provide me with those .npy files. Thank you very much.
